docs/tempo/website/getting-started/_index.md

```diff
@@ -1,16 +1,19 @@
 ---
-title: Getting started
+title: Get started
+menuTitle: Get started with Grafana Tempo
 weight: 150
 ---
 
-# Getting started 
+# Get started with Grafana Tempo
 
 Distributed tracing visualizes the lifecycle of a request as it passes through
-a set of applications. There are a few components that must be configured in order to get a
+a set of applications.
+There are a few components that must be configured in order to get a
 working distributed tracing visualization.
 
 This document discusses the four major pieces necessary to build out a tracing system:
-Instrumentation, Pipeline, Backend and Visualization. If one were to build a diagram laying
+Instrumentation, Pipeline, Backend and Visualization.
+If one were to build a diagram laying
 out these pieces, it may look something like this:
```

Comment:
```suggestion
The following diagram illustrates a tracing system configuration:
```

Reply:
Good catch. I read this and thought passive voice, should be changed. Thank you.